src/lib/y2storage/proposal_settings.rb

```diff
@@ -441,20 +442,39 @@ def load_features
       load_encryption
     end
 
+    def load_encryption_hash(enc)
+      passwd = enc["password"]
+      self.encryption_password = passwd if !passwd.nil? && !passwd.empty?
+
+      type = enc["type"]
```

Comment:
Being a bit nitpicking, I would not use "type" as key for that entry of the hash. You are using it to set the value of `Encryption#encryption_method`, which is fully correct. But there is another attribute `Encryption#type` which has a different set of values and can be easily be confused.